microraiden/microraiden/test/fixtures/web3.py

```diff
@@ -192,7 +212,15 @@ def channel_manager_proxy_factory(privkey):
             GAS_PRICE, GAS_LIMIT,
             use_tester
         )
-    return channel_manager_proxy_factory
+    # Everything is deployed. Make snapshot.
+    if use_tester:
+        snapshot_id = web3.testing.snapshot()
+
+        yield channel_manager_proxy_factory
+
+        web3.testing.revert(snapshot_id)
```

Comment:
This doesn't make sense. The chain will be reverted when all tests finish?

Reply:
Damn, didn't see the `session` scope. I guess I will make a `web3_snapshot` fixture that both `channel_manager` and `client` use. That explains why the tests are slower.

I had the issue that the channel manager reverted the tester chain before the client could clean up its channels. So I'll look into fixing that again.